src/App.jsx

```diff
@@ -23,14 +23,14 @@ import { MESSAGE, TYPE } from "./constants";
 
 function App() {
   const dispatch = useDispatch();
-  const history = useHistory();
+  const navigate = useNavigate();
```

Comment:
https://reactrouter.com/docs/en/v6/upgrading/v5#use-usenavigate-instead-of-usehistory